Dangerfile

```diff
@@ -128,3 +154,7 @@ suggested_labels = labelsForModifiedFiles()
 if !suggested_labels.empty?
   addLabels(suggested_labels)
 end
+
+# Warn if the PR contains DO NOT SUBMIT. Don't fail here, since it's still nice
+# to run travis on WIP pull requests.
+warn("PR contents contain 'do not submit'.") if do_not_submit
```

Comment:
This is going to become a mess. Could we group the issuing of the warning with the code that does the checking?

Could we make these different checks functions and put things like `do_not_submit` into locals within a function?

Reply:
I'll look into ways to make this file less spaghetti.